### PR TITLE
[Issue 132]: Add stacked layout option when showing all cells.

### LINKED ISSUE #132: Cell > Dashboard > * new option
Currently, there are two options:

* Show All Cells - densely packs all cells with output into a dashboard layout
* Hide All Cells - hides all cells

We can satisfy @jtyberg's requirement originally documented in #55 by having 3 options:

* Show All Cells (Packed) - does what it does today
* Show All Cells (Top-to-Bottom) - shows all cells, full width, in notebook order
* Hide All Cells - does what it does today

### PR DESCRIPTION
**Pull request by Lull3rSkat3r**
Fixes #132.

### CODE REVIEW

**Comment by parente** [comment]
Works as advertised.

File: urth_dash_js/notebook/dashboard-view/sub-menu.html [hunk 1]
```diff
@@ -3,10 +3,14 @@
     <a href="#">Dashboard</a>
     <ul class="dropdown-menu">
         <li id="urth-dashboard-show-all" class="disabled" title="Return all hidden cells to the Dashboard">
-            <a href="#">Show All Cells</a>
+            <a href="#">Show All Cells (Packed)</a>
+        </li>
+        <li id="urth-dashboard-show-all-stacked" class="disabled" title="Return all hidden cells to the Dashboard">
```
Comment (jhpedemonte) [inline urth_dash_js/notebook/dashboard-view/sub-menu.html]: You should update the `title` attribute to differentiate between the two layout strategies. These show when you hover over menu items.

**Comment by parente** [comment]
@jhpedemonte Good catch. Totally missed the tooltips.

@Lull3rSkat3r can you submit another PR with it?

**Comment by Lull3rSkat3r** [comment]
@parente, PR #136

**Pull request merged**